lib/users.test.js

```diff
@@ -13,17 +20,20 @@ const genUserInfoWithPw = async (password) => {
 
 const mockFindOne = jest.fn()
 const mockCreateAccount = jest.fn()
+const mockUpdate = jest.fn()
 db.getModels = () => {
     return {
         Accounts: {
             findOne: mockFindOne,
-            create: mockCreateAccount
+            create: mockCreateAccount,
+            update: mockUpdate
         }
     }
 }
 
 const signUp = users.signUp
 const logIn = users.logIn
+const setDBPassword = users.setDBPassword
```

Comment:
This could all be done on the same line using destructuring. 
`const {signUp, logIn, setDBPassword} = users